### PR TITLE
Performance support for Serverless on Node

### PR DESCRIPTION
**Pull request by ajjindal**

### CODE REVIEW

File: src/platforms/node/guides/aws-lambda/index.mdx [hunk 1]
```diff
@@ -70,7 +68,8 @@ exports.handler = Sentry.AWSLambda.wrapHandler(yourHandler, {
 
 With the AWS Lambda integration enabled, the Node SDK will:
 
-- Automatically report all exceptions from your lambda functions
+- Automatically report all events from your Lambda Functions.
+- You can <PlatformLink to="/configuration/filtering/#filtering-transaction-events">modify the transaction sample rate</PlatformLink> using `tracesSampleRate`.
```
Comment (PeloWriter) [inline src/platforms/node/guides/aws-lambda/index.mdx:72]:
```suggestion
- Allows you to <PlatformLink to="/configuration/filtering/#filtering-transaction-events">modify the transaction sample rate</PlatformLink> using `tracesSampleRate`.
```

File: src/platforms/node/guides/gcp-functions/index.mdx [hunk 2]
```diff
@@ -4,44 +4,69 @@ redirect_from:
...
+```
 
 Check out Sentry's [GCP sample apps](https://github.com/getsentry/examples/tree/master/gcp-cloud-functions/node) for detailed examples. Refer to the [JavaScript docs](/platforms/javascript/) for more configuration options.
+
+## Behavior
+
+With the AWS Lambda integration enabled, the Node SDK will:
+
+- Automatically report all events from your Lambda Functions.
+- You can <PlatformLink to="/configuration/filtering/#filtering-transaction-events">modify the transaction sample rate</PlatformLink> using `tracesSampleRate`.
```
Comment (PeloWriter) [inline src/platforms/node/guides/gcp-functions/index.mdx:63]:
```suggestion
- Allows you to <PlatformLink to="/configuration/filtering/#filtering-transaction-events">modify the transaction sample rate</PlatformLink> using `tracesSampleRate`.
```

File: src/platforms/node/guides/gcp-functions/index.mdx [hunk 3]
```diff
@@ -4,44 +4,69 @@ redirect_from:
...
+- Issue reports automatically include:
+  - A link to the cloudwatch logs
+  - Function details
+  - sys.argv for the function
+  - AWS Request ID
+  - Function execution time
+  - Function version
+- Sentry holds the thread for upto 2 seconds to report errors. You can change flush time limit by defining a `flushTimeout` value in the handler options
```
Comment (PeloWriter) [inline src/platforms/node/guides/gcp-functions/index.mdx:71]:
```suggestion
- Sentry holds the thread for up to two seconds to report errors. You can change flush time limit by defining a `flushTimeout` value in the handler options
```

**Review by PeloWriter — APPROVED** [review]

File: src/platforms/node/guides/aws-lambda/index.mdx [hunk 4]
```diff
@@ -79,3 +78,4 @@ With the AWS Lambda integration enabled, the Node SDK will:
   - Function execution time
   - Function version
 - Sentry holds the thread for upto 2 seconds to report errors. You can change flush time limit by defining a `flushTimeout` value in the handler options
+- Request data is attached to all events: **HTTP method, URL, headers, form data, JSON payloads**. Sentry excludes raw bodies and multipart file uploads. Sentry also excludes personally identifiable information (such as user ids, usernames, cookies, authorization headers, IP addresses) unless you set `sendDefaultPii` to `True`.
```
Comment (kamilogorek) [inline src/platforms/node/guides/aws-lambda/index.mdx:81]: There's no `sendDefaultPii` in JS SDKs

File: src/platforms/node/guides/gcp-functions/index.mdx [hunk 5]
```diff
@@ -4,44 +4,69 @@ redirect_from:
   - /platforms/node/gcp_functions/
 ---
 
-Add `@sentry/node` as a dependency:
+_(New in version 5.26.0)_
 
-```bash {tabTitle:npm}
-npm install --save @sentry/node
-```
+Add `@sentry/serverless` as a dependency to `package.json`:
```
Comment (kamilogorek) [inline src/platforms/node/guides/gcp-functions/index.mdx:9]: Why we use `package.json` instructions here, but `npm install` in AWSLambda? They should be consistent

File: src/platforms/node/guides/gcp-functions/index.mdx [hunk 6]
```diff
@@ -4,44 +4,69 @@ redirect_from:
...
+- Allows you to <PlatformLink to="/configuration/filtering/#filtering-transaction-events">modify the transaction sample rate</PlatformLink> using `tracesSampleRate`.
+- Issue reports automatically include:
+  - A link to the cloudwatch logs
+  - Function details
+  - sys.argv for the function
+  - AWS Request ID
+  - Function execution time
+  - Function version
+- Sentry holds the thread for up to two seconds to report errors. You can change flush time limit by defining a `flushTimeout` value in the handler options
+- Request data is attached to all events: **HTTP method, URL, headers, form data, JSON payloads**. Sentry excludes raw bodies and multipart file uploads. Sentry also excludes personally identifiable information (such as user ids, usernames, cookies, authorization headers, IP addresses) unless you set `sendDefaultPii` to `True`.
```
Comment (kamilogorek) [inline src/platforms/node/guides/gcp-functions/index.mdx:72]: Same here. There's no `sendDefaultPii`.

File: src/platforms/node/guides/gcp-functions/index.mdx [hunk 7]
```diff
@@ -4,44 +4,69 @@ redirect_from:
...
+exports.helloEvents = Sentry.GCPFunction.wrapCloudEventFunction((context, callback) => {
+  throw new Error('oh, hello there!');
+});
+```
 
 Check out Sentry's [GCP sample apps](https://github.com/getsentry/examples/tree/master/gcp-cloud-functions/node) for detailed examples. Refer to the [JavaScript docs](/platforms/javascript/) for more configuration options.
+
+## Behavior
+
+With the AWS Lambda integration enabled, the Node SDK will:
```
Comment (kamilogorek) [inline src/platforms/node/guides/gcp-functions/index.mdx:60]: Copy paste from Lambda 🙃

File: src/wizard/node/gcpfunctions.md [hunk 8]
```diff
@@ -5,40 +5,49 @@ support_level: production
 type: framework
 ---
 
-Add `@sentry/node` as a dependency:
+Add `@sentry/serverless` as a dependency to `package.json`:
```
Comment (kamilogorek) [inline src/wizard/node/gcpfunctions.md:8]: Same here. Inconsistent instructions.

**Review by kamilogorek — CHANGES_REQUESTED** [review]

File: src/platforms/node/guides/gcp-functions/index.mdx (hunk 5 above)
Comment (ajjindal) [reply src/platforms/node/guides/gcp-functions/index.mdx:9]: GCP pulls in npm modules from package.json without having to install Sentry.

**Pull request merged**